### PR TITLE
Tracefoss arc42 documentation

### PR DESCRIPTION
**Pull request by tbraun-zf**
arc42 documentation related to [TRACEFOSS-757](https://jira.catena-x.net/browse/TRACEFOSS-757)

### CODE REVIEW

File: docs/src/docs/arc42/introduction-goals/quality-goals.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+= Quality goals
```
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/quality-goals.adoc:1]: I would recommend to mention this topics as well 

- Security 
- application  Reliability 
- UUX 
- Performance
- cloud agnostic solution
- reference application

File: docs/src/docs/arc42/introduction-goals/stakeholder.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
+= Stakeholder
+
+The following table presents the stakeholders of Trace-X and their respective intentions.
+
+|===
+|Who? |Matters and concern
```
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/stakeholder.adoc:6]:
```suggestion
|Who |Matters and concern
```

File: docs/src/docs/arc42/introduction-goals/requirements.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+== What is the purpose of the Trace-X application
+* Empower all companies from SMEs to large OEMs to participate in parts traceability with an Open-Source solution to easily take part in Catena-X Traceability Use Case.
+* It is a standalone application which can be self-hosted.
+* Display the relations of the automotive value chain based on a standardized IT model.
+* Overview and transparency across the supplier network enable faster intervention based on recorded events in the supply chain.
+* Notifications/Massages regarding Quality related incidents and a tool for inspecting the supply chain.
+
+== Essential features
+* List and view manufactured parts based on BoM AsBuild
```
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/requirements.adoc:11]: is BoM asPlanned as well an essential feature?

File: docs/src/docs/arc42/introduction-goals/requirements.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+* Show detailed information on manufactured parts from Aspects
+* Asset Administration Shell description
+* Submodels SerialPartTypization, AssemblyPartRelationship, Batch
+* List and view Supplier parts (AssemblyPartRelationship) based on BoM AsBuild lifecycle
+* View parts and parts relations in a visualized parts tree
+* Send and receive top-down notifications (quality investigations) along the supply chain
```
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/requirements.adoc:17]: Should we mention upstream visibility as well?

File: docs/src/docs/arc42/introduction-goals/requirements.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+* Empower all companies from SMEs to large OEMs to participate in parts traceability with an Open-Source solution to easily take part in Catena-X Traceability Use Case.
+* It is a standalone application which can be self-hosted.
+* Display the relations of the automotive value chain based on a standardized IT model.
+* Overview and transparency across the supplier network enable faster intervention based on recorded events in the supply chain.
+* Notifications/Massages regarding Quality related incidents and a tool for inspecting the supply chain.
+
+== Essential features
+* List and view manufactured parts based on BoM AsBuild
+* Show detailed information on manufactured parts from Aspects
+* Asset Administration Shell description
```
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/requirements.adoc:13]: Building graph structure based on aas description assets

File: docs/src/docs/arc42/introduction-goals/requirements.adoc [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+* Submodels SerialPartTypization, AssemblyPartRelationship, Batch
```
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/requirements.adoc:14]: SingleLevelUsageAs* Built and Planned as other important submodels.

File: docs/src/docs/arc42/introduction-goals/requirements.adoc (hunk 4 above)
Comment (mkanal) [inline docs/src/docs/arc42/introduction-goals/requirements.adoc:17]: quality alerts is not part of the essential features list.

File: docs/src/docs/arc42/introduction-goals/requirements.adoc (hunk 3 above)
Comment (tbraun-zf) [reply docs/src/docs/arc42/introduction-goals/requirements.adoc:11]: Not implemented yet.

File: docs/src/docs/arc42/introduction-goals/requirements.adoc (hunk 4 above)
Comment (tbraun-zf) [reply docs/src/docs/arc42/introduction-goals/requirements.adoc:17]: Not implemented yet.

File: docs/src/docs/arc42/introduction-goals/requirements.adoc (hunk 5 above)
Comment (tbraun-zf) [reply docs/src/docs/arc42/introduction-goals/requirements.adoc:13]: Not implemented yet.

File: docs/src/docs/arc42/introduction-goals/requirements.adoc (hunk 6 above)
Comment (tbraun-zf) [reply docs/src/docs/arc42/introduction-goals/requirements.adoc:14]: Not implemented yet.

File: docs/src/docs/arc42/introduction-goals/quality-goals.adoc (hunk 1 above)
Comment (tbraun-zf) [reply docs/src/docs/arc42/introduction-goals/quality-goals.adoc:1]: Relevant bullets introduced with new commit.

File: docs/src/docs/arc42/introduction-goals/requirements.adoc (hunk 5 above)
Comment (tbraun-zf) [reply docs/src/docs/arc42/introduction-goals/requirements.adoc:13]: Reworked within new commit.

**Review by dblankevia — APPROVED** [review]

**Pull request merged**